ci/steps.yml

```diff
@@ -12,7 +12,7 @@ steps:
   ## we insert sleep in random duration < 30sec to reduce
   ## download server load.
   - bash: |
-      set -e
+      set -ex
```

Comment:
This turns on `xtrace`, which prints the values of all the variable substitutions in the Azure Pipelines logs. I couldn't figure out any other way to debug this file other than to do this. I can go back and take these out if you prefer.